From 83e6510a024da955db32638bd17cbbf4545ded5f Mon Sep 17 00:00:00 2001
From: "vikas.kumar4" <vikas.kumar4@BLR1-MAC-N03254.local>
Date: Tue, 10 Jul 2018 12:24:38 +0530
Subject: [PATCH 1/2] added 'Hello' source string

---
 myfile.js | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/myfile.js b/myfile.js
index 095fda2..4d0c2ef 100644
--- a/myfile.js
+++ b/myfile.js
@@ -27,5 +27,6 @@ Mkt3L10n.lobEditor = {
     Description: "Description",
     your_noteson_this_form: "your notes on this form",
     Layout: "Layout",
-    How_the_labels_and_fields_are_arranged_in_this_form: "How the labels and fields are arranged in this form"
+    How_the_labels_and_fields_are_arranged_in_this_form: "How the labels and fields are arranged in this form",
+    Hello: "Hello"
 };  
\ No newline at end of file
-- 
2.15.1 (Apple Git-101)

